WtWSMS/common/culture/names_equivalency/00_names.txt

```diff
@@ -2158,7 +2124,6 @@ jamila_female = { Jamila Jamilah }
 jamne_female = { Jamne }
 jamne_male = { Jamne }
 jamshid_male = { CemS_I_d Cems_i_d Jamshid }
-januarius_male = { Zennaru }
```

Comment:
Why did you remove this?

Reply:
> Why did you remove this?

Duplicated equivalence. 

> Why did you remove this?

Same as above, name moved to ianuarius_male.